jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/TiffModule.java

```diff
@@ -941,26 +934,24 @@ protected void checkValidity(TiffIFD ifd, RepInfo info)
             int[] colormapBlueValue = niso.getColormapBlueValue();
             if (colormapBitCodeValue == null || colormapRedValue == null
                     || colormapGreenValue == null || colormapBlueValue == null) {
-                reportInvalid("ColorMap not defined for " + "palette-color",
+                reportInvalid(MessageConstants.ERR_COL_MAP_NOT_DEF,
                         info);
             }
             if (samplesPerPixel != 1) {
-                reportInvalid("For palette-color SamplesPerPixel "
-                        + "must be 1: " + samplesPerPixel, info);
+                reportInvalid(MessageConstants.ERR_PAL_COL_SPP_NE_1 + samplesPerPixel, info);
             }
             int len = (1 << bitsPerSample[0]);
             if (colormapBitCodeValue.length < len) {
-                reportInvalid("Insufficient ColorMap values for "
-                        + "palette-color: " + colormapBitCodeValue.length + "<"
-                        + len, info);
+                reportInvalid(MessageConstants.ERR_COL_MAP_MISS_VALS +
+                		      colormapBitCodeValue.length + "<"  + len,
+                		      info);
```

Comment:
Mixed tabs and spaces. Does Eclipse not have a "use existing indentation style"-type setting? If not, I can't recommend IntelliJ enough, for that and many other quality-of-life improvements. One of my own life's few regrets is trying to stick with Eclipse for as long as I did while IntelliJ was an option.